internal/system/agent/executor/operation.go

```diff
@@ -45,6 +46,10 @@ func NewOperations(
 // operationViaExecutor delegates a start/stop/restart operation request to the configuration-defined executor.
 func (e operations) Do(services []string, operation string, parameters []string) interface{} {
 	var result []interface{}
+	if len(services) == 0 || operation == "" {
+		return fmt.Sprintf("incorrect or malformed body was passed in with the request")
+	}
+
```

Comment:
Is this the same behavior as Edinburgh?  

To ensure backwards contract compatibility, we should implement the same behavior and return the same error message as Edinburgh.

Reply:
This is actually _not_ the same behavior as Edinburgh. There was no such error-handling being done in Edinburgh.

Reply:
What is the behavior of Edinburgh when invalid data is posted -- what does Edinburgh implementation return as the response body?  We want to duplicate that behavior to ensure backwards compatibility.